project/CodeCoverageSettings.scala

```diff
@@ -0,0 +1,22 @@
+import sbt.Setting
+import scoverage.ScoverageKeys
+
+object CodeCoverageSettings {
+
+  private val excludedPackages: Seq[String] = Seq(
+    "<empty>",
+    "Reverse.*",
+    ".*BuildInfo.",
+    "uk.gov.hmrc.BuildInfo",
+    ".*Routes",
+    ".*RoutesPrefix*",
+    ".*definition*"
+  )
+
+  val settings: Seq[Setting[?]] = Seq(
+    ScoverageKeys.coverageExcludedPackages := excludedPackages.mkString(";"),
+    ScoverageKeys.coverageMinimumStmtTotal := 80,
+    ScoverageKeys.coverageFailOnMinimum := true,
+    ScoverageKeys.coverageHighlighting := true
+  )
+}
```

Comment:
I don't like this (generally, I prefer to have everything in the build file for an easier overview). Will we get warnings about deprecated keys if we put it in a separate file?

Reply:
It's the standard approach for all new projects and we are doing it elsewhere in Clipper. We do still get warnings about deprecated keys